app/src/main/java/com/ikarabulut/Rules.java

```diff
@@ -4,24 +4,17 @@ public class Rules {
 
     private static boolean isAliveNextGeneration;
 
-    public static boolean aliveCheck(int aliveNeighborsCount) {
+    public static boolean determineFate(int aliveNeighborsCount, boolean isAlive) {
```

Comment:
Similar to above comment, "determineFate" makes me think it's going to return "alive" or "dead", but if I asked the Rules "what's the fate of this cell?" and it responded with "true", that'd be confusing. So I think this should really be named something like "shouldBeAliveNextGeneration" OR you could change all your booleans like this to return "alive" or "dead", but obviously that'd be a much larger architectural change. I'm just saying that to illustrate my point for the most part.

Overall I think of fate/status to be alive/dead, but you're mostly using booleans, so then most names should become things like "isAlive" or "shouldBeAlive".